activerecord/lib/active_record/associations/preloader.rb

```diff
@@ -103,7 +103,7 @@ def call
 
         loaders = build_preloaders
         group_and_load_similar(loaders)
-        loaders.map(&:run)
+        loaders.each(&:run)
```

Comment:
This had me scratching my head for a while as we've just called run on the loaders, but I guess it's needed as we don't have a `ThroughAssociations.load_records_in_batch` method yet. Makes sense!